deeplabcut/lmot/main.py

```diff
@@ -0,0 +1,122 @@
+import os
+import os.path
+import sys
+import time
+from pathlib import Path
+
+import cv2
+import numpy as np
+import tensorflow as tf
+from skimage.util import img_as_ubyte
+
+from deeplabcut.lmot import mot
+from deeplabcut.lmot.assets import bbox
+from deeplabcut.pose_estimation_tensorflow.config import load_config
+from deeplabcut.pose_estimation_tensorflow.nnet import predict
+from deeplabcut.utils import auxiliaryfunctions
+
+
+def analyze_image(config, vid, shuffle=1, trainingsetindex=0, gputouse=None, save_as_csv=False):
```

Comment:
could this be the original "image analysis" code instead? https://github.com/AlexEMG/DeepLabCut/blob/da7851d633de118a6bf6e9709805cc113b58f87b/deeplabcut/pose_estimation_tensorflow/predict_videos.py#L618